doc/content/integrations/cloud-integrations/aws-iot/default/things.md

```diff
@@ -17,7 +17,7 @@ Create and claim things only for devices that are not yet in your {{% tts %}} ap
 
 The AWS IoT Integration for {{% tts %}} uses the thing that you specified when deploying the integration. By default, the thing type is `lorawan`.
 
-Things are created automatically as they join the network or send an uplink message. For the name of new things, you can use their `<DevEUI>`
+Things are created automatically as they join the network or send an uplink message. For the name of new things, you can use their `<DevEUI>` or `<stack-name>_<device-id>`.
```

Comment:
you should probably add a note here about what stack name and what device id, or at least link the explanation you added in the `deployment-guide.md` file